apollo-router/src/configuration/mod.rs

```diff
@@ -576,6 +577,28 @@ impl Default for Supergraph {
 pub(crate) struct Apq {
     /// Cache configuration
     pub(crate) experimental_cache: Cache,
+
+    pub(crate) subgraph: ApqSubgraphWrapper,
+}
+
+/// Configuration options pertaining to the subgraph server component.
+#[derive(Debug, Clone, Default, Deserialize, Serialize, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub(crate) struct ApqSubgraphWrapper {
```

Comment:
I'd like to make a generic wrapper for this, since we'll use it in other places, but it looks like it will require a bit more work than the scope of this PR https://serde.rs/attr-bound.html